src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/NetworkWatcherAPITests.ps1

```diff
@@ -402,7 +372,7 @@ function Test-PacketCapture
     # Setup
     $resourceGroupName = Get-ResourceGroupName
     $nwName = Get-ResourceName
-    $location = "eastus"
+    $location = "centraluseuap"
```

Comment:
@irrogozh it would be nice if these tests used the `Get-Location` function (from our common test code), which will allow you to get a valid location for a provided resource type, as well as give it a "priority location" that it will return if available.

You can see how Compute does this for their different resource types [here](https://github.com/Azure/azure-powershell/blob/daa377500761c183d86d6463b54db2ca52603f15/src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/ComputeTestCommon.ps1#L661). In your case, you would be making the following call:

```powershell
Get-Location "Microsoft.Network" "networkWatchers" "centraluseuap"
```